webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -222,7 +222,7 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
           ...experimentFilters,
           filter: filtersString,
           limit: isPagedView ? settings.pageLimit : 2 * PAGE_SIZE,
-          offset: isPagedView ? Math.max(page - 1, 0) * settings.pageLimit : tableOffset,
```

Comment:
Question: what was the intention of `Math.max(page - 1, 0)`?

Reply:
I think previously when `page` is undefined in `pagedView` it would work out to be `0` which causes the offset to be a negative number.